keps/sig-node/3673-kubelet-parallel-image-pull-limit/README.md

```diff
@@ -612,12 +632,18 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
+This is an opt-in feature, and it does not change any default behavior. We manually tested enabling and disabling this feature by changing kubelet config and
```

Comment:
Manual test is fine, but please describe the test in more detail.
[https://github.com/kubernetes/enhancements/pull/3658/files is one of the nice examples]